### ISSUE TITLE
Combined logger pod selector.

### ISSUE THREAD

**Issue by shanna** [GoogleContainerTools/skaffold#666]
In my project I have pods that are started with skaffold but don't have build artifacts such as my traefik ingress controller I'd love to be able to see the logging for in the combined logs. It would be helpful if you could configure skaffold with selectors to log an entire application label or namespace not just the build artifact pods.

Prior art:
* https://github.com/bakins/kube-log-tail takes a selector though it suffers issues when pods are replaced/restarted.

**Comment by r2d4** [comment]
+1

Not sure how the UX on this looks - possibly another top level `log` map?


```
apiVersion: skaffold/v1alpha2
kind: Config
build:
  artifacts:
  - imageName: gcr.io/k8s-skaffold/skaffold-example
deploy:
  kubectl:
    manifests:
      - k8s-*
log:
  labels:
    app: redis
```

**Comment by shanna** [comment]
With the https://github.com/GoogleContainerTools/skaffold/pull/644/files label PR merged logging could default to `{"deployed-with": "skaffold"}` in the watch config. The log config would simply override the label and `ImageList` removed.

**Comment by r2d4** [comment]
Thats a great idea @shanna. The code has been merged now and this is the log code that selects pods based on their name

https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/kubernetes/log.go#L93

Since we just added label support, WDYT of making this an OR statement "podSelector || podLabelSelector"? It should catch everything that we currently catch, in addition to anything else deployed with skaffold.
